pkg/handler/collector/github/github.go

```diff
@@ -0,0 +1,204 @@
+//
+// Copyright 2022 The GUAC Authors.
```

Comment:
2023 for new files!

Reply:
> me, still living in 2020: 
<img src="https://user-images.githubusercontent.com/68356865/211879209-892cab11-43c4-448e-8d25-b5b976c5b359.png" height="200" width="250" />